docs/usecases.md

```diff
@@ -0,0 +1,126 @@
+
+How this should be done within 15 minutes (not days or weeks)?<br>
+How do you ensure you really removed this user everywhere?
+
+### Solution
+
+#### Don't
+1. Have users installed on the systems itself.
+2. Share passwords of shared users or even the `root` user.
+3. Have user's public keys stored at shared users or even the `root` user.
+
+#### Do
+Use the [OpenID Connect authorization](reference/authorization/oidc.md).
+
+As the users always authorized by your [Identity Provider (IdP)](https://openid.net/developers/how-connect-works/), and this is always evaluated when someone tries to access the service via SSH, it will also immediately reject the authorization to this service.
+
+No need to access any of these services directly to remove/de-authorize these users.
+
+If the [environments are configured accordingly](reference/environment/index.md) (which is the default) all user's files and processes will be removed/killed automatically, too.
+
+## On-board users within 15 minutes in the organization {:id=onboard}
+
+This is quite similar to [Off-board users within 15 minutes of the organization](#offboard), but obviously reverse.
+
+### Problem
+
+1. Assume you're part of an organization.
+2. Assume this organization has more than _just_ 10 people who might be able to access SSH resources.
+3. Assume you've to on-board an employee, now.
+4. Assume you should ensure that this employee is ab to have access to all services, now.
+
+In cases of SSH servers, this often results in running through all servers and either:
+
+* Share the server shared-user passwords,
+* Add user's public key to a shared user,
+* Add a dedicated user (with password or authorized key),
+* or change the [Ansible](https://www.ansible.com/) or [Puppet](https://www.puppet.com/) configuration apply it at every machine.
+
+How this should be done now (not days or weeks)?<br>
+"Did I really give him access everywhere?"
+
+### Solution
+
+Use the [OpenID Connect authorization](reference/authorization/oidc.md).
+
+There is no need to create them somewhere on the server itself. The [OIDC authorization](reference/authorization/oidc.md) will resolve them using the configured [Identity Provider (IdP)](https://openid.net/developers/how-connect-works/) - that's it!
+
+No need to access any of these services directly to create/authorize these users.
+
+If the [environments are configured accordingly](reference/environment/index.md) (which is the default) all user's resources (like home directory) will be created automatically.
+
+## Bastion Host / Jump Host {:id=bastion}
+
+### Problem
+
+1. Assume you've to manage resources.
+2. These resources are not directly reachable for you. They are protected inside other networks, you have no direct access to. For example, you're sitting at home and there is a another service inside an [AWS private VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).
```

Comment:
These resources are not directly accessible to you. They are protected within other networks to which you have no direct access. For example, you're sitting at home and there's another service inside an [AWS private VPC]. (https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).